sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/implementation/helpers/Utility.java

```diff
@@ -0,0 +1,26 @@
+package com.azure.maps.render.implementation.helpers;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.azure.maps.render.implementation.models.BoundingBoxPrivate;
+import com.azure.maps.render.implementation.models.IncludeText;
+import com.azure.maps.render.models.BoundingBox;
+import com.azure.maps.render.models.Center;
+
+public class Utility {
+    public static BoundingBoxPrivate toBoundingBoxPrivate(BoundingBox boundingBox) {
+        BoundingBoxPrivate boundingBoxPrivate = new BoundingBoxPrivate();
```

Comment:
Definitely not needed. This is only needed to convert private -> public.

Here you just need to set the properties on boundingboxprivate directly. Are they accessible? are there setters?